pkgs/development/nim-packages/packedjson/default.nix

```diff
@@ -0,0 +1,8 @@
+{ fetchFromGitHub }:
+
+fetchFromGitHub {
+  owner = "Araq";
+  repo = "packedjson";
+  rev = "7198cc8";
+  sha256 = "1ay2zd88q8hvpvigsg8h0y5vc65hk3lk0d48fy9hwg4lcng19mp1";
+}
```

Comment:
those fetcher only packages have no maintainer nor license

Reply:
Yes, but for now not a regression as these dependencies were completely hidden.

Reply:
fair enough